src/mlpack/methods/ann/layer/c_relu.hpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file c_relu_impl.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of CReLU layer.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_C_RELU_HPP
+#define MLPACK_METHODS_ANN_LAYER_C_RELU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ *
+ * A concatenated ReLU has two outputs, one ReLU and one negative ReLU,
+ * concatenated together.In other words, for positive x it produces [x, 0],
+ * and for negative x it produces [0, x].Because it has two outputs,
+ * CReLU doubles the output dimension.
+ *
+ * Note:
+ * During building of model, The next layer of crelu should have double the size
```

Comment:
This sound strange, what about "The CReLU doubles the output size.". And remove the rest?

Reply:
Changed